internal/staticanalysis/obfuscation/utils/transform.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+func Transform[T, R any](ts []T, fn func(T) R) []R {
+	result := make([]R, len(ts))
+	for i, t := range ts {
+		result[i] = fn(t)
+	}
+	return result
+}
+
+func TransformMap[K comparable, V, R any](m map[K]V, fn func(K, V) R) []R {
```

Comment:
Is this function used anywhere? Is it worth keeping?

The behavior seems unusual to me. My expectation would be that returned a map, rather than a slice.

Reply:
Will remove in another PR